internal/lookout/repository/util.go

```diff
@@ -50,6 +50,16 @@ func StartsWith(field exp.IdentifierExpression, pattern string) goqu.Expression
 	return field.Like(pattern + "%")
 }
 
+// Do a glob (sql like) search if the pattern contains any * characters,
+// otherwise do an exact match.
+func GlobSearchOrExact(field exp.IdentifierExpression, pattern string) goqu.Expression {
```

Comment:
Is this a special thing to do because wildcards are special in SQL?

Reply:
Part of the idea is speed. If the user hasn't indicated they want a wider match, we default to an exact match, which should be a faster query. 

The use of `*` is just inertia, I doubt most users want to use `%` to denote a wildcard.

Reply:
Ah TIL.  Should we support % as a wildcard also then?

Reply:
I don't think so. I don't think I've ever used it as such outside of SQL. But I'm willing to be persuaded. @JamesMurkin any thoughts?

Reply:
I'm not sure users would expect % to be a wildcard and I'm not sure we gain much out of allowing multiple - although not massively harmful

I personally think we should stick to just using *, but it wouldn't be terrible if we wanted to support both

Reply:
I'm not sure users would expect % to be a wildcard and I'm not sure we gain much out of allowing multiple - although not massively harmful

I personally think we should stick to just using *, but it wouldn't be terrible if we wanted to support both

Reply:
Sounds good to me.